rust/ql/lib/codeql/files/FileSystem.qll

```diff
@@ -44,7 +45,9 @@ class File extends Container, Impl::File {
     result =
       count(int line |
         exists(AstNode node, Location loc |
-          not node instanceof SourceFile and loc = node.getLocation()
+          not node instanceof Module and
```

Comment:
Please revert this change. The module isn't the problem. The problem are the `/* */` comments which syntactically belong to the `mod`, `struct`, `macrorule`, etc. that follows them. Just verify for yourself using the AST viewer.

Reply:
Hmm, I see what you mean now, `Module` is a module decl not a module.  I'm going to have to get used to that.

Reply:
I think the AST viewer works fine on test databases these days. I just built a database running `codeql test run --keep-databases`, loaded it in vscode, ran `from File f select f`, click on a file, and right-click "view AST" in that file.

Reply:
That method for tests is a bit of a fiddle unfortunately, but good to know the AST viewer now works for Rust.  I had investigated using an adhoc query to print all the `AstNode.toString()`s associated with each line of each file, which worked - though I didn't get a hierarchy.

Anyway I've created an issue for the locations problems.